pkgs/development/libraries/audio/zita-resampler/default.nix

```diff
@@ -15,7 +15,10 @@ stdenv.mkDerivation rec {
 
   patchPhase = ''
     cd source
-    sed -e "s@ldconfig@@" -i Makefile
+    substituteInPlace Makefile \
+      --replace 'ldconfig' "" \
+      --replace '-march=native' "" \
+      --replace '-DENABLE_SSE2' ""
```

Comment:
I think it is better to keep this flag on `x86_64` systems.

Reply:
Certainly sse2 should be present for all x86_64 machines, but I was thinking that the compiler would set this automatically. Thinking again, I'm not sure this is a standard flag so should probably turn it back on for x86_64 :+1:

Reply:
Yeah, this looks like a macro flag that should be used by the code internally.